Jenkinsfile

```diff
@@ -220,13 +228,14 @@ pipeline {
                               } else if (ARCH=="KNL" && PARTYPE=="SEQ") {
                                 sh """
                                   source /etc/profile.d/modules.sh
+                                  export SLURM_CONF=$HOME/slurm.conf
                                   export OMP_NUM_THREADS=10
                                   export I_MPI_PMI_LIBRARY=/usr/lib64/libpmi.so
                                   while : ; do
                                     set +e
-                                    output=`srun -n 1 --time=00:05:00 ./src/${it.join('-')} $legacyCellProcessorOption -t -d ./test_input/ 2>&1`
+                                    output=`srun --jobid=$knl_jobid -n 1 --time=00:05:00 ./src/${it.join('-')} $legacyCellProcessorOption -t -d ./test_input/ 2>&1`
```

Comment:
will fail if job is not yet allocated, e.g. because the cluster is already full. so you have to wait for the job here.
(until knl_jobstate is running or so)